shell-script/useless-vm-check.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# ENVIRONMENT VARIABLES:
+# 
+#   * AWS_ACCESS_KEY, required, the AWS access key
+#   * AWS_SECRET_KEY, required, the AWS secret key
+#   * GITHUB_TOKEN, required, GitHub API authentication token
+set -euo pipefail
+AWS_ACCESS_KEY=${AWS_ACCESS_KEY_ID}
+AWS_SECRET_KEY=${AWS_SECRET_ACCESS_KEY}
+GITHUB_TOKEN=${GITHUB_TOKEN}
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+function markdown_report() {
+    case "$OSTYPE" in
+        darwin*)
+            if ! type gsed &> /dev/null || ! type ggrep &> /dev/null; then
+                echo "GNU sed and grep not found! You can install via Homebrew" >&2
+                echo >&2
+                echo "    brew install grep gnu-sed" >&2
+                exit 1
+            fi
+
+            SED=gsed
+            GREP=ggrep
+            ;;
+        *)
+            SED=sed
+            GREP=grep
+            ;;
+    esac
+    echo "**Daily VM check List**:"
+    echo "| Tag | Instance_id | state |"
+    echo "| :--- | :--- | :--- |"
+    cat ${GITHUB_WORKSPACE}/instances.txt
```

Comment:
Can it add more detail about the resources? E.g. region, created-time, resource link.

Reply:
done